pkg/analyzer/endpoints/blocks.go

```diff
@@ -0,0 +1,28 @@
+package endpoints
+
+import (
+	"github.com/np-guard/models/pkg/netset"
+)
+
+// a base struct to represent external endpoints, segments and rule block
+type ipBlock struct {
+	Block *netset.IPBlock
+	name  string
+}
+type RuleBlock struct {
+	ipBlock
+	VMs []EP
```

Comment:
the VMs field is not used currently? can it be removed?

Reply:
I do not collect the VMs yet, but the code I use it at 
getEndpointsFromGroupsPaths()